content/kots-cli/upstream.md

```diff
@@ -5,10 +5,22 @@ title: kots upstream
 weight: 90180
 ---
 
-The `upstream` command interacts with a KOTS application that is installed in an Admin Console.
+The `kots upstream upgrade` fetches the latest version of the upstream application. It is functionality equivalent to clicking the Check For Updates in the Admin Console. 
 
-```shell
-kubectl kots upstream upgrade --namespace <namespace> <app-slug>
+### Usage
+```bash
+kubectl kots upstream upgrade [upstream-uri] [flags]
```

Comment:
does this also require namespace

Reply:
namespace is included in the [flags]. Strictly speaking, not a required argument, so I left it out of the usage in this case.

Reply:
(Note that the: "kubectl kots upstream --help" only returns the "help for upstream" flag).